src/client/driver/command-executors/execute-action-command.js

```diff
@@ -148,6 +152,15 @@ function ensureCommandElements (command, timeout) {
         });
 }
 
+function ensureCommandArguments (command) {
+    if (command.type === COMMAND_TYPE.pressKey) {
+        var parsedKeySequence = parseKeySequence(command.keys);
```

Comment:
I was thinking about this. Technically we can do it but we should "share" several client scripts between the server and client and split the `testcafe/src/client/runner/utils/key-maps.js` file into the client-only part (because it uses browser's user agent) and the shared part.